### PR TITLE
galaxy-data: initial commit

### PR DESCRIPTION
**Pull request by bernt-matthias**
TODO:

- [x] galaxy-objectstore https://github.com/bioconda/bioconda-recipes/pull/26366
- [x] isa-rwval https://github.com/bioconda/bioconda-recipes/pull/26774
- [ ] openidconnect?

Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by bgruening** [comment]
> FileNotFoundError: [Errno 2] No such file or directory: 'test-requirements.txt'

@natefoo we need also here a new version

File: recipes/galaxy-data/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{% set version = "20.9.0" %}
+{% set sha256 = "0cc55cefbf0434660a20916b2e0060abcc41550504f48b21311711d4bc5a21c8" %}
```
Comment (natefoo) [inline recipes/galaxy-data/meta.yaml:2]:
```suggestion
{% set version = "20.9.1" %}
{% set sha256 = "d078a651b22dbd9ccc68a3cfd991d12fbf539dcfc941115242c0c4c734ef8527" %}
```

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Comment by natefoo** [comment]
Does this perhaps need an explicit dependency on OpenSSL?

```
% singularity exec /cvmfs/singularity.galaxyproject.org/all/galaxy-data:20.9.1--py_0 python -m pysam
WARNING: Skipping mount /opt/singularity-3.7.1/var/singularity/mnt/session/etc/resolv.conf [files]: /etc/resolv.conf doesn't exist in container
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/local/lib/python3.6/runpy.py", line 142, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/usr/local/lib/python3.6/runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "/usr/local/lib/python3.6/site-packages/pysam/__init__.py", line 5, in <module>
    from pysam.libchtslib import *
ImportError: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory
```

**Comment by bernt-matthias** [comment]
I guess this is because https://github.com/bioconda/bioconda-recipes/issues/17212 .. maybe be should force `pysam >0.14`?

**Comment by bernt-matthias** [comment]
Wondering why conda chooses 0.14 ..

**Comment by natefoo** [comment]
Ah yes that seems to be the issue.

**Pull request merged**